hands-on-exercise/5-ibc-adv/5-ibc-app-packets.md

```diff
@@ -1,8 +1,8 @@
 ---
 title: "Adding Packet and Acknowledgement Data"
 order: 6
-description: 
-tags: 
+description:
```

Comment:
shouldn't add empty fields - let's set or remove

Reply:
Ok I expect this is on prupose as we have a few more sections with that, to overwrite the default?